requirements.txt

```diff
@@ -1,6 +1,6 @@
 lz4
 xxhash
 numpy
-Cython>=0.22
+Cython>=0.22,!=0.27.1
```

Comment:
I updated to 0.27.3 and still got the following compilation error:

``` 
Error compiling Cython file:
------------------------------------------------------------
...
                                    self.index[startpos,1],
                                    pybuf.len) == bkey:
                                return True
                            startpos += 1
                finally:
                    PyBuffer_Release(cython.address(pybuf))
                                          ^
------------------------------------------------------------

sharedbuffers/mapped_struct.py:4055:43: Taking address of non-lvalue (type Python object)
```

Reply:
Seems like a different error. Will look into it on monday

Reply:
Ok, I will create a new ticket for this error.

Reply:
It seems like another bug in Cython